src/extra/AutoUI/index.tsx

```diff
@@ -268,10 +268,12 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 
 	return (
 		<Flex
+			height="100%"
 			flexDirection={Array.isArray(data) ? 'column' : undefined}
 			mt={Array.isArray(data) ? 2 : undefined}
 		>
 			<Spinner
+				height="100%"
```

Comment:
I think it also works with v right?
```suggestion
				flex={1}
```